packages/core/test/types/hooks.ts

```diff
@@ -158,6 +159,18 @@ sequelize.addHook('beforeConnect', (...args) => {
   expectTypeOf(args).toMatchTypeOf<[ConnectionOptions]>();
 });
 
+sequelize.beforePoolAcquire('name', (options?: GetConnectionOptions) => {
```

Comment:
Could you add these for `afterPoolAcquire` as well?

Reply:
Done 😊